lib/raw.js

```diff
@@ -72,14 +72,32 @@ module.exports = function() {
       await Promise.all(_.flatMap(dat.entries, ref => {
         return _.map(['input', 'output', 'schema'], type => {
           if (ref[type]) {
-            let sname = path.basename(url.parse(ref[type]).pathname);
+            let sname = ref[type].startsWith('http://') ?
+              path.basename(url.parse(ref[type]).pathname) :
+              ref[type].replace(/#$/, '');
```

Comment:
Maybe some comments here to help us figure out what this means later?  I think the second alternative is the "new" style, right?